### DISCUSSION TITLE
npx create-turbo@latest is not working

### DISCUSSION THREAD

**Discussion by piyush1008** [vercel/turborepo#8967] [Help] [answered]
### Summary

running npx create-turbo@latest and using npm workspaces to install the dependancies is giving error 

>>> no "package.json" found at /Users/piyushdubey/Documents/web-dev/mono/my-turborepo

**Comment by anthonyshew** [comment, marked as answer]
This happened because we named the repository.

Fixed now! Thanks for flagging!
